docs/chat/chat.rst

```diff
@@ -1,30 +1,25 @@
-.. _chat_page:
-
 ==============
-LLM Playground
+Chat Page
 ==============
 
-LLM Playground
---------------
+.. _chat:
 
-Starting from version 24.03, Backend.AI introduces the 'LLM Playground' feature, 
-which enables users to conveniently compare and interact with different LLM models all in one location.
+Starting from version 25.05, the 'LLM Playground' feature has been separated into its own page and renamed to Chat. 
+Chat page enables users to conveniently compare and interact with different LLM models all in one location.
 This allows users to experience the services offered by Backend.AI as well as a variety of large language models (LLMs).
 
-
-.. image:: LLM_playground.png
-    :align: center
-    :alt: LLM Playground
+.. image:: chat_page.png
+    :alt: chat page
 
 Selecting models
 ----------------
 
-Each card's top left corner features an endpoint selector and a model selector. The endpoint selector 
-enables the search and selection of endpoints established by Backend.AI services. The model selector allows for the choice of a model ID; 
-If the model is unavailable, it defaults to 'Custom'.
+You can select the endpoint and model from the top left corner of each chat card on the Chat page. 
```

Comment:
```suggestion
Users can select the endpoint and model from the top left corner of each chat card on the Chat page. 
```
Please don't use 'You'. Instead of that, use 'user(s)' or 'It'.